requirements.yml

```diff
@@ -47,3 +47,6 @@ collections:
     type: git
   - name: https://github.com/ovirt/ovirt-ansible-collection
     type: git
+  - name: https://github.com/linux-system-roles/timesync.git
```

Comment:
Please also add this to `galaxy.yml` under `dependencies`

Reply:
done!

Reply:
I spoke too soon...

I'm using a single role because I don't need the entire collection but in the `dependency` section of galaxy.yml you can only define collections.  Well, since linux-system-roles don't have a single repo to handle the collection (they rely on a [script](https://github.com/linux-system-roles/auto-maintenance#release_collectionpy) for creating the release) so in any case i need to define it in its "galaxy" format.

And _obviously_ collection version is different from role version.

cc @pablintino @cjeanner

Reply:
Ugh, supper annoying!

What you have done seems right to me, install the role as that's what we use and as we can't define a dependency to the single role we use, define it to `linux-system-roles` as it's still correct.